src/rdkafka.h

```diff
@@ -2957,6 +2973,22 @@ rd_kafka_position (rd_kafka_t *rk,
  * `rd_kafka_topic_new()`.
  *
  * `rd_kafka_produce()` is an asynch non-blocking API.
+ * See `rd_kafka_conf_set_dr_msg_cb` on how to setup a callback to be called
+ * once the delivery status (success or failure) is known.
```

Comment:
add: "The delivery report callback is triggered by the application calling rd_kafka_poll() (at regular intervals) or rd_kafka_flush() (at termination)."